packages/notebook-extension/style/index.css

```diff
@@ -0,0 +1,6 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+@import url(~@jupyterlab/cells/styles/index.css);
```

Comment:
This brings up an interesting question.  Should the extensions (and the examples) import all of the CSS that they need, or should we have each package import its direct dependencies (notebook requires cell, cell requires output area).

The tradeoff is convenience vs. flexibility.